README.md

```diff
@@ -315,6 +315,9 @@ output: extended
 
 # Set a default receiver
 receiver: team-X-pager
+
+# Skip TLS certificate verification
+tls.insecure.skip.verify: true
```

Comment:
Let's get this out of the example configuration to keep it simple and not promote insecure settings.